new pg-query-stream publish is missing the dist folder

3.4.0 (published 4hrs ago) was published without the dist folder, which causes this error.

```
Cannot find module '/node_modules/pg-query-stream/dist/index.js'. Please verify that the package.json has a valid "main" entry
```

--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
     "build": "tsc --build",
     "build:watch": "tsc --build --watch",
     "pretest": "yarn build",
+    "prepublish": "yarn build",
     "lint": "eslint '*/**/*.{js,ts,tsx}'"
   },
   "devDependencies": {

